test

```diff
@@ -3,135 +3,155 @@ set -ex
 
 DOMAIN_SOCKET="$(pwd)/mina-indexer.sock"
 
-# Create temporary directory for blocks and things in a cross platform
-# compatible way
 TMPDIR=$(mktemp -d 2>/dev/null || mktemp -d -t 'mytmpdir')
 BLOCKS_DIR="${TMPDIR}/blocks"
-mkdir $BLOCKS_DIR
+mkdir "${BLOCKS_DIR}"
 
 cleanup() {
     echo "Exiting with $?"
-    rm -rf $TMPDIR
-}
-trap cleanup EXIT
-
-cargo build
-
-teardown() {
-    rm -rf $HOME/.mina-indexer/database || true
-    rm -rf $HOME/.mina-indexer/logs || true
+    rm -rf ${TMPDIR} 2>/dev/null
```

Comment:
To be clear, it's the surrounding double quotes, not the surrounding braces, that save you from having the TMPDIR being interpreted as multiple directories. So, this is still not safe, FYI.